shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -0,0 +1,58 @@
+- 8:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    os_variant = rhel8
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 4096
```

Comment:
Is this really required? I'm using 1.5G for standard and 2G for url installation in my CI and it works well... (or is it mentioned somewhere as a minimum? Do you have a link to documentation?)

Reply:
I do not know. I have copied this bit from `7.devel.cfg` file. If 2G works for you, we can set this to be 2G. It was commited here https://github.com/avocado-framework/avocado-vt/commit/d95909de1f38220fe1e542bdd9eebab769752bda with commit message snippet: `..installing RHEL over
http fails with memory set to 1024.`. I do not know why increase memory to 2048 was not enough.

Reply:
yes, please. 2G should suffice... (I have one machine with only 4G which might be problematic. I know, I can override the default, but...)